doc/docportal/settings/game.rst

```diff
@@ -16,12 +16,11 @@ All settings can also be changed in the :doc:`../advanced_topics/configuration_f
 
 ,,,,,,,
 
-.. _gameid:
+.. _id:
 
 **ID**
 	Shows the short name of the game. This ID can be used for launching the game from the command line.
```

Comment:
please add: "You can change it at your convenience, but only letters, numbers and dashes (-) are allowed"